string name
float64 newHarga
---
string harga_baru